### ISSUE TITLE
ES6 and NPM packages

### ISSUE THREAD

**Issue by hantsy** [shawnmclean/Idle.js#26]
Hi @shawnmclean Thanks for providing IdleJS. 

I am trying to integrate Idle in our applications, but I encountered some problems. We are using ES6/Webpack.

I download a copy of `idle.js`, when import it , none of the following are working.

```
import './idle';
import Idle from './idle';
import * as Idle from './idle';
```

Can you provide some suggestion?

BTW, I can not find this lib in NPM, it is not published to NPM?

**Comment by dietergeerts** [comment]
Yes, can this be added to npm for easier inclusion in other projects? Thx.
